recipes/yasm/all/conanfile.py

```diff
@@ -1,9 +1,15 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+from conan import ConanFile
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.build import cross_building
+from conan.tools.files import chdir, copy, download, get, mkdir, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.layout import basic_layout, vs_layout
+from conan.tools.microsoft import is_msvc, MSBuildDeps, MSBuildToolchain, MSBuild, VCVars
+
 import os
-import shutil
 
-required_conan_version = ">=1.33.0"
 
+required_conan_version = ">=1.52.0"
```

Comment:
too high

Reply:
This is the one documented here: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/autotools_package/all/conanfile.py#L11

How can I determine the minimum conan version this recipe is compatible with ?

Reply:
theres a partial cheatsheet https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#import-tools-from-conan

Reply:
The version requirement is good since we use `conan.tools.files.export_conandata_patches` introduced in 1.52.0.